ml-agents-envs/mlagents/envs/subprocess_env_manager.py

```diff
@@ -106,11 +121,15 @@ def _send_response(cmd_name, payload):
             elif cmd.name == "external_brains":
                 _send_response("external_brains", env.external_brains)
             elif cmd.name == "reset_parameters":
-                _send_response("reset_parameters", env.reset_parameters)
+                reset_params = {}
+                for k in shared_float_properties.list_properties():
```

Comment:
Why not just make a get_properties() method that returns a dict? (same in SimpleEnvManager.reset_parameters property)